Gems/Atom/RHI/Vulkan/Code/Source/RHI/Scope.cpp

```diff
@@ -23,7 +23,7 @@
 #include <RHI/SwapChain.h>
 #include <RHI/Framebuffer.h>
 #include <RHI/Device.h>
-#include <Atom/RHI.Reflect/Vulkan/Conversion.h>
```

Comment:
`Conversion.h` needs to be exposed in `Atom/RHI.Reflect/Vulkan` because it needs to be accessible outside Atom by other gems which use Vulkan's `RHI.Reflect` target. [OpenXRVk](https://github.com/o3de/o3de-extras/blob/6de5c12c77a5cdabdbf402a4ab1a08766afe9aa3/Gems/OpenXRVk/Code/Source/OpenXRVkSwapChain.cpp#L9) gem uses it. Please make sure this doesn't break OpenXRVk.

Reply:
Thanks for confirming, I can see that `ConvertFormat` is still part of `Atom/RHI.Reflect/Vulkan/Conversion.h`.